common/src/main/java/com/wynntils/core/persisted/config/Config.java

```diff
@@ -135,6 +136,10 @@ public <E extends Enum<E>> T tryParseStringValue(String value) {
             return (T) EnumUtils.fromJsonFormat((Class<E>) getType(), value);
         }
 
+        if (get() instanceof CustomColor) {
```

Comment:
You will need to wait for https://github.com/Wynntils/Wynntils/pull/3490, and then remove this.